<?xml version="1.0"?>
<launch>
    <!--### ASUQTR CONTROL NODE ###-->
    <node pkg="asuqtr_control_node" type="control_node.py" name="control_node" output="screen" >
        <rosparam command="load" file="$(find asuqtr_control_node)/params/actuators_mock_signal.yaml" />
        <rosparam command="load" file="$(find asuqtr_control_node)/params/cost_matrix.yaml" />
        <rosparam command="load" file="$(find asuqtr_control_node)/params/game_pad_parameters.yaml" />
        <rosparam command="load" file="$(find asuqtr_control_node)/params/limits_n_thresholds.yaml" />
        <rosparam command="load" file="$(find asuqtr_control_node)/params/lqr_enabled_on_start.yaml" />
    </node>

    <!--### ROSBRIDGE NODE FOR WEB PAGE ###-->
    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>

</launch>
